recipes-core/u-boot/u-boot%.bbappend

```diff
@@ -35,5 +34,8 @@ do_prepcompile() {
 	for frag in ${UBOOT_ENV_SRC_FRAGS}; do
 		cat ${WORKDIR}/$frag >> ${WORKDIR}/${UBOOT_ENV_SRC}
 	done
+
+	# override hardcoded sources
+	cp ${WORKDIR}/${UBOOT_ENV_SRC} ${WORKDIR}/boot.cmd
```

Comment:
do you remember in what case we had to do this? this comment is not reminding me enough :P

Reply:
We need to ask to @ricmm

I never knew why he added that :)

Reply:
ok i added a better comment after reading the meta-sunxi source... imo that one should be improved to just use UBOOT_ENV_SRC as source and not the hardcoded boot.cmd. maybe worth checking and trying to upstream